zero-thrs-control/src/thrs/input_output/modules/thrusters.py

```diff
@@ -44,91 +41,103 @@ class ThrustersSensorValues(ThrsModel):
         sensor.FlowSensor, component_meta(yard_tag="50001057-23")
     ]
     thrusters_flowcontrol_aft: Annotated[
-        sensor.Valve, component_meta(yard_tag="5001064-01", valve_type="flowcontrol")
+        sensor.Valve, component_meta(yard_tag="50001215", valve_type="flowcontrol")
     ]
     thrusters_flowcontrol_fwd: Annotated[
-        sensor.Valve, component_meta(yard_tag="5001064-02", valve_type="flowcontrol")
+        sensor.Valve, component_meta(yard_tag="50001064-02", valve_type="flowcontrol")
     ]
     thrusters_shutoff_recovery: Annotated[
-        sensor.Valve, component_meta(yard_tag="5001069-10", valve_type="shutoff")
+        sensor.Valve, component_meta(yard_tag="50001066-03", valve_type="shutoff")
     ]
     thrusters_switch_aft: Annotated[
         sensor.Valve, component_meta(yard_tag="50001091-01", valve_type="switch")
     ]
     thrusters_switch_fwd: Annotated[
         sensor.Valve, component_meta(yard_tag="50001091-02", valve_type="switch")
     ]
-    thrusters_flow_recovery_aft: Annotated[
-        sensor.FlowSensor, component_meta(yard_tag="5001093-01")
-    ]
-    thrusters_flow_recovery_fwd: Annotated[
-        sensor.FlowSensor, component_meta(yard_tag="5001093-02")
+    thrusters_flow_recovery: Annotated[
+        sensor.FlowSensor, component_meta(yard_tag="50001093-01")
     ]
-    thrusters_pressure_recovery: Annotated[
+    thrusters_pressure_pump: Annotated[
         sensor.PressureSensor, component_meta(yard_tag="50001097-01")
     ]
-    thrusters_pressure_cooling: Annotated[
+    thrusters_pressure_relief: Annotated[
         sensor.PressureSensor, component_meta(yard_tag="50001097-02")
     ]
-
     thrusters_aft: Annotated[
-        sensor.Thruster, component_meta(yard_tag="15001001", included_in_fmu=False)
+        sensor.Thruster, component_meta(yard_tag="150001001", included_in_fmu=False)
     ]
     thrusters_fwd: Annotated[
-        sensor.Thruster, component_meta(yard_tag="15001002", included_in_fmu=False)
+        sensor.Thruster, component_meta(yard_tag="150001002", included_in_fmu=False)
     ]
     thrusters_pcs: Annotated[
         sensor.Pcs, component_meta(yard_tag="1500", included_in_fmu=False)
     ]
 
+    @computed_field
+    @property
+    def thrusters_temperature_recovery(
+        self,
+    ) -> Annotated[
+        sensor.TemperatureSensor, component_meta(included_in_fmu=False)
+    ]:
+        total_flow = self.thrusters_flow_aft.flow.value + self.thrusters_flow_fwd.flow.value
+        if total_flow > 0.0:
+            averaged_return_temperature = (
+                self.thrusters_temperature_aft_return.temperature.value * self.thrusters_flow_aft.flow.value
+                + self.thrusters_temperature_fwd_return.temperature.value * self.thrusters_flow_fwd.flow.value
+            ) / total_flow
+        else:
+            averaged_return_temperature = 0.0  # TODO: 0 ok?
+
+        return sensor.TemperatureSensor(temperature=Stamped.stamp(averaged_return_temperature))
```

Comment:
I think this should have the `min` or `max` of `temp_aft, temp_fwd, flow_aft, flow_fwd`